### PR TITLE
Resubmit instead of constraint error if gas bumping would exceed maximum

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

File: CHANGELOG-core.md [hunk 1]
```diff
@@ -17,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - ETH_DISABLED flag works again
 
+
```
Comment (topliceanu) [inline CHANGELOG-core.md:24]: remove this whitespace to keep the changeset short

File: core/store/tx_manager.go [hunk 2]
```diff
@@ -685,14 +685,15 @@ func (txm *EthTxManager) handleSafe(
 }
 
 func (txm *EthTxManager) BumpGasByIncrement(originalGasPrice *big.Int) *big.Int {
-	return BumpGas(txm.config, originalGasPrice)
+	gasPrice, _ := BumpGas(txm.config, originalGasPrice)
```
Comment (topliceanu) [inline core/store/tx_manager.go:688]: Should this error be logged!

**Review by topliceanu — COMMENTED** [review]
lgtm

File: core/store/tx_manager.go (hunk 2 above)
Comment (samsondav) [reply core/store/tx_manager.go:688]: Its for the legacy tx manager which isnt used any more.

I just havent got around to deleting the code yet.

**Review by topliceanu — APPROVED** [review]

**Review by se3000 — APPROVED** [review]

**Pull request merged**